src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -830,15 +831,15 @@ class module_elf : public module_impl
         if (auto search = arg2patcher.find(key_string); search != arg2patcher.end())
           search->second.m_ctrlcode_patchinfo.emplace_back(patcher::patch_info{ctrlcode_offset, 0, 0});
         else
-          arg2patcher.emplace(std::move(key_string), patcher{ symbol_type, {{ctrlcode_offset, 0}}, buf_type});
+          arg2patcher.emplace(key_string, patcher{symbol_type, {{ctrlcode_offset, 0}}, buf_type});
```

Comment:
key_string should be moved., why was this changed?, maybe because key_string was made `const`  which was not quite right.
```suggestion
          arg2patcher.emplace(std::move(key_string), patcher{symbol_type, {{ctrlcode_offset, 0}}, buf_type});
```